lib/traffic_source_parser/result/search.rb

```diff
@@ -1,12 +1,27 @@
 require 'traffic_source_parser/result/generic'
+require 'traffic_source_parser/channels/types/paid'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.